PWGGA/GammaConvBase/AliAODConversionPhoton.h

```diff
@@ -132,6 +148,3 @@ class AliAODConversionPhoton : public AliAODConversionParticle, public AliConver
 
 
```

Comment:
Please increase the classdef here. Otherwise the PR looks fine.

Reply:
Thanks for catching, classdef is incremented now